tests/conftest.py

```diff
@@ -58,8 +52,8 @@
 @pytest.fixture
 def config():
     return {
-        "http": {"hostname": "127.0.0.1", "port": "6680"},
-        "proxy": {"hostname": "host_mock", "port": "port_mock"},
+        "http": {"hostname": "127.0.0.1", "port": 6680},
+        "proxy": {"hostname": "host_mock", "port": 80},
```

Comment:
Not that it matters much, but 8080 is probably a better example value for an HTTP proxy.

```suggestion
        "proxy": {"hostname": "host_mock", "port": 8080},
```